### PR TITLE
Add addq and deleteq

### PR DESCRIPTION
**Pull request by ketamethane**
BUG: In `MainWindow`, if lines 120 and 121 are uncommented/used in class of lines 117 and 118, then the app throws a `NullPointerException` when initialised.
Note: Current implementation only shows Question list, Person List is commented out.

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
+import seedu.address.logic.commands.*;
```
Comment (Shawn532) [inline src/main/java/seedu/address/logic/parser/AddressBookParser.java:9]: I am not sure, but I think this kind of import won't pass the Java CI.

File: src/main/java/seedu/address/model/AddressBook.java [hunk 2]
```diff
@@ -158,11 +167,71 @@ public ObservableList<Tutorial> getTutorialList() {
     public boolean equals(Object other) {
         return other == this // short circuit if same object
                 || (other instanceof AddressBook // instanceof handles nulls
-                && persons.equals(((AddressBook) other).persons));
+                && persons.equals(((AddressBook) other).persons)
+                && questions.equals(((AddressBook) other).questions));
```
Comment (Shawn532) [inline src/main/java/seedu/address/model/AddressBook.java:171]: Wow nice. I forgot to change this in my code.

**Review by Shawn532 — COMMENTED** [review]
It looks good overall. I only found several typos. I haven't finished the UI and Storage part myself, so I am not very sure how those parts work. I can review that again after finishing those parts myself.

File: src/main/java/seedu/address/logic/commands/AddQCommand.java [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+import static seedu.address.logic.parser.CliSyntax.PREFIX_NAME;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PHONE;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_TAG;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.question.Question;
+
+/**
+ * Adds a person to the address book.
```
[the remark this answers is not in the document]
Comment (ketamethane) [reply src/main/java/seedu/address/logic/commands/AddQCommand.java:15]: yup, modified!

File: src/main/java/seedu/address/logic/commands/DeleteQCommand.java [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+        this.targetIndex = targetIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Question> lastShownList = model.getFilteredQuestionList();
+
+        if (targetIndex.getZeroBased() >= lastShownList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
```
[the remark this answers is not in the document]
Comment (ketamethane) [reply src/main/java/seedu/address/logic/commands/DeleteQCommand.java:39]: changed!

File: src/main/java/seedu/address/logic/parser/AddQCommandParser.java [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+
+import seedu.address.logic.commands.AddQCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+import seedu.address.model.question.Description;
+import seedu.address.model.question.Question;
+
+/**
+ * Parses input arguments and creates a new AddCommand object
```
[the remark this answers is not in the document]
Comment (ketamethane) [reply src/main/java/seedu/address/logic/parser/AddQCommandParser.java:11]: i overlooked this. added!

File: src/main/java/seedu/address/logic/parser/AddressBookParser.java (hunk 1 above)
Comment (ketamethane) [reply src/main/java/seedu/address/logic/parser/AddressBookParser.java:9]: resolved!

**Pull request merged**